1.x-centos7/Dockerfile

```diff
@@ -4,13 +4,13 @@ MAINTAINER mail@racktear.com
 RUN groupadd tarantool \
     && adduser -g tarantool tarantool
 
-ENV TARANTOOL_VERSION=1.7.6-11-gcd17b77f9 \
+ENV TARANTOOL_VERSION=1.9.1-11-gcd17b77f9 \
     TARANTOOL_DOWNLOAD_URL=https://github.com/tarantool/tarantool.git \
     TARANTOOL_INSTALL_LUADIR=/usr/local/share/tarantool \
     LUAROCKS_URL=https://github.com/tarantool/luarocks/archive/6e6fe62d9409fe2103c0fd091cccb3da0451faf5.tar.gz \
     LUAROCK_SHARD_REPO=https://github.com/tarantool/shard.git \
     LUAROCK_SHARD_TAG=8f8c5a7 \
-    LUAROCK_AVRO_SCHEMA_VERSION=2.0.1 \
```

Comment:
@sharonovd So, you install the last version of avro-schema?

Reply:
oh lord